src/components/table/README.md

```diff
@@ -901,6 +901,65 @@ event will not be emitted when the input, select, textarea is clicked (unless th
 (even when disabled)
 
 
+## Row select support
+You can make rows selectable, by using the prop `selectable`.
+
+Users can easily change the selecting mode by setting the `select-mode` prop.
+ - `multi`: each click will select/deselect the row (default mode)
+ - `single`: only a single row can be selected at one time
+ - `range`: any row clicked is toggled, any other deselected. the SHIFT key selects a range of
+rows, and CTRL/CMD click will toggle the selected row.
+
+When a table is `selectable` and the user clicks on a row, `<b-table>` will emit the `row-selected`
+event, passing a single argument which is the complete list of selected items.
+**Treat this argument as read-only.**
+
+**Note:** _Paging or sorting will clear the selection._
+
+```html
+<template>
+  <b-form-group
+      label="Selection mode:"
+      horizontal
+      :label-cols="4"
+      breakpoint="md"
+  >
+    <b-form-select v-model="selectMode" :options="modes" class="mb-3" />
+  </b-form-group>
+  <b-table selectable :select-mode="selectMode" selectedVariant="success" :items="items" @row-selected="rowSelected"></b-table>
+  {{ selected }}
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      modes: [
+        { value: 'multi', text: 'multi' },
+        { value: 'single', text: 'single' },
+        { value: 'range', text: 'range' }
```

Comment:
You could just pass a simple array of string modes, since both text and value are the same (b-select with magically convert the simple array of strings to the array of objects form)

Reply:
yes but I was wondering if you'd prefer to change the label

Reply:
I think the label (option text) should match the actual mode value. Was just thinking that a simple text array would shorten the code.